wrangler.toml

```diff
@@ -1,4 +1,4 @@
-name = "discord-slash-commands"
+name = "hello-bot"
```

Comment:
Why was this changed? Just a query

Reply:
changed according to my bot,  reverted back, thanks for pointing out

Reply:
Please change back the name